kalite/topic_tools/settings.py

```diff
@@ -12,16 +12,22 @@
 import os
 from django.conf import settings
 
+CHANNEL = getattr(settings, "CHANNEL", "khan")
```

Comment:
Yes, but I explicitly changed all references to CHANNEL to only come from
topic tools settings.

On Thu, 17 Sep 2015 12:46 Michael Gallaspy <notifications@github.com> wrote:

> In kalite/topic_tools/settings.py
> <https://github.com/learningequality/ka-lite/pull/4440#discussion_r39790186>
> :
>
> > @@ -12,16 +12,22 @@
> >  import os
> >  from django.conf import settings
> >
> > +CHANNEL = getattr(settings, "CHANNEL", "khan")
>
> If the user doesn't specify CHANNEL in their settings module, could the
> following situation be problematic?
>
> from django.conf import settingsfrom kalite.topic_tools import settings as tt_settings
>
> settings.CHANNEL  # AttributeError
> tt_settings.CHANNEL  # "khan"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/4440/files#r39790186>.
>